packages/odyssey-stylelint/src/index.js

```diff
@@ -0,0 +1,260 @@
+        'animation',
+        'animation-name',
+        'animation-duration',
+        'animation-play-state',
+        'animation-timing-function',
+        'animation-delay',
+        'animation-iteration-count',
+        'animation-direction',
+        'border',
+        'border-top',
+        'border-right',
+        'border-bottom',
+        'border-left',
+        'border-width',
+        'border-top-width',
+        'border-right-width',
+        'border-bottom-width',
+        'border-left-width',
+        'border-style',
+        'border-top-style',
+        'border-right-style',
+        'border-bottom-style',
+        'border-left-style',
+        'border-radius',
+        'border-top-left-radius',
+        'border-top-right-radius',
+        'border-bottom-right-radius',
+        'border-bottom-left-radius',
+        'border-color',
+        'border-top-color',
+        'border-right-color',
+        'border-bottom-color',
+        'border-left-color',
+        'outline',
+        'outline-color',
+        'outline-offset',
+        'outline-style',
+        'outline-width',
+        'stroke-width',
+        'stroke-linecap',
+        'stroke-dasharray',
+        'stroke-dashoffset',
+        'stroke',
+        'opacity',
+        'background',
+        'background-attachment',
+        'background-clip',
+        'background-color',
+        'background-image',
+        'background-repeat',
+        'background-position',
+        'background-size',
+        'box-shadow',
+        'fill',
+        'color',
+        'font',
+        'font-family',
+        'font-size',
+        'font-size-adjust',
+        'font-smoothing',
+        'font-stretch',
+        'font-style',
+        'font-variant',
+        'font-weight',
+        'font-emphasize',
+        'font-emphasize-position',
+        'font-emphasize-style',
+        'letter-spacing',
+        'line-height',
+        'list-style',
+        'text-align',
+        'text-align-last',
+        'text-decoration',
+        'text-decoration-color',
+        'text-decoration-line',
+        'text-decoration-style',
+        'text-indent',
+        'text-justify',
+        'text-overflow',
+        'text-overflow-ellipsis',
+        'text-overflow-mode',
+        'text-rendering',
+        'text-outline',
+        'text-shadow',
+        'text-transform',
+        'text-wrap',
+        'word-wrap',
+        'word-break',
+        'text-emphasis',
+        'text-emphasis-color',
+        'text-emphasis-style',
+        'text-emphasis-position',
+        'vertical-align',
+        'white-space',
+        'word-spacing',
+        'hyphens',
+        'src',
+        'tab-size',
+        'counter-reset',
+        'counter-increment',
+        'resize',
+        'cursor',
+        'pointer-events',
+        'speak',
+        'user-select',
+        'nav-index',
+        'nav-up',
+        'nav-right',
+        'nav-down',
+        'nav-left'
+      ],
+      { unspecified: 'bottom' }
+    ],
```

Comment:
I'd like to propose removing these 214 lines of property order configuration for something that is simpler to reason about, less work to maintain, and easier to adopt by other teams in the future such as alpha ordering.